includes/admin/payments/orders.php

```diff
@@ -784,26 +882,39 @@ function edd_order_details_extras( $order = false ) {
 
 				<?php if ( edd_is_add_order_page() ) : ?>
 					<div class="edd-order-tx-id edd-admin-box-inside">
-						<label for="edd_transaction_id" class="label"><?php esc_html_e( 'Transaction ID', 'easy-digital-downloads' ); ?></label>
-						<input type="text" name="transaction_id" id="edd_transaction_id" value="" />
+						<div class="edd-form-group">
+							<label for="edd_transaction_id" class="edd-form-group__label label"><?php esc_html_e( 'Transaction ID', 'easy-digital-downloads' ); ?></label>
+							<div class="edd-form-group__control">
+								<input type="text" name="transaction_id" class="edd-form-group__input" id="edd_transaction_id" value="" />
+							</div>
+						</div>
 					</div>
 				<?php endif; ?>
 
 				<div class="edd-unlimited-downloads edd-admin-box-inside">
-					<label for="edd_unlimited_downloads" class="label label--has-tip label--has-checkbox">
-						<input type="checkbox" name="edd-unlimited-downloads" id="edd_unlimited_downloads" value="1"<?php checked( true, $unlimited, true ); ?>/>
-						<?php esc_html_e( 'Unlimited Downloads', 'easy-digital-downloads' ); ?>
-						<span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>Unlimited Downloads</strong>: checking this box will override all other file download limits for this purchase, granting the customer unliimited downloads of all files included on the purchase.', 'easy-digital-downloads' ); ?>"></span>
-					</label>
+					<div class="edd-form-group">
+						<div class="edd-form-group__control">
+							<input type="checkbox" name="edd-unlimited-downloads" id="edd_unlimited_downloads" class="edd-form-group__input" value="1"<?php checked( true, $unlimited, true ); ?>/>
+
+							<label for="edd_unlimited_downloads">
```

Comment:
Should this label and the one for the send receipt checkbox have the `__label` class added? I realize that class has the display set to `block`, but the CSS could be updated to something like this for consistency:

```css
input + label.edd-form-group__label {
    display: inline;
}
```

Reply:
@robincornett My thinking is that `.edd-form-group__label` applies _only_ to the item being used as the label for the control itself. `label`s inside of `edd-form-group__control` are not the label for the form group, they are just arbitrary markup inside of the control.